frontend/src/views/shop/DetailShop.vue

```diff
@@ -0,0 +1,903 @@
+                            >
+                                <icon-fa6-solid:shield-halved /> Admin
+                            </a>
+                        </dd>
+                    </div>
+                </dl>
+                <div
+                    class="mt-6 sm:mt-0 sm:col-span-1 sm:row-span-full sm:col-start-2 sm:row-start-1
+                     md:col-start-3 md:row-span-full md:row-start-1"
+                >
+                    <img
+                        v-if="shopStore.shop.shopImage"
+                        :src="`/${shopStore.shop.shopImage}`"
+                        class="h-[200px] sm:h-[400px] md:h-[200px]"
+                    >
+                    <icon-fa6-solid:image
+                        v-else
+                        class="text-gray-100 text-9xl"
+                    />
+                </div>
+            </div>
+        </div>
+
+        <tabs
+            :labels="[
+                { key: 'checks', title: 'Checks', icon: FaCircleCheck },
+                { key: 'extensions', title: 'Extensions', icon: FaPlug },
+                { key: 'tasks', title: 'Scheduled Tasks', icon: FaListCheck },
+                { key: 'queue', title: 'Queue', icon: FaCircleCheck },
+                { key: 'pagespeed', title: 'Pagespeed', icon: FaRocket },
+                { key: 'changelog', title: 'Changelog', icon: FaFileWaverform }
+            ]"
+        >
+            <template #panel-checks>
+                <data-table
+                    :columns="[
+                        { key: 'message', name: 'Message' },
+                    ]"
+                    :data="shopStore.shop.checks || []"
+                >
+                    <template #cell-message="{ row }">
+                        <icon-fa6-solid:circle-xmark
+                            v-if="row.level == 'red'"
+                            class="text-red-600 mr-2 text-base dark:text-red-400 "
+                        />
+                        <icon-fa6-solid:circle-info
+                            v-else-if="row.level === 'yellow'"
+                            class="text-yellow-400 mr-2 text-base dark:text-yellow-200 "
+                        />
+                        <icon-fa6-solid:circle-check
+                            v-else
+                            class="text-green-400 mr-2 text-base dark:text-green-300"
+                        />
+                        <a
+                            v-if="row.link"
+                            :href="row.link"
+                            target="_blank"
+                        >
+                            {{ row.message }}
+                            <icon-fa6-solid:up-right-from-square class="text-xs" />
+                        </a>
+                        <template v-else>
+                            {{ row.message }}
+                        </template>
+                    </template>
+                    <template #cell-actions="{ row }">
```

Comment:
Action label is gone
<img width="1235" alt="Bildschirmfoto 2024-03-09 um 18 50 14" src="https://github.com/FriendsOfShopware/shopmon/assets/412142/27cf98ec-f352-48d8-9662-b0aec3a8d00d">
<img width="1232" alt="Bildschirmfoto 2024-03-09 um 18 50 26" src="https://github.com/FriendsOfShopware/shopmon/assets/412142/d7ddd826-57d5-4d0a-b71c-ede42a51cbe8">

<img width="1239" alt="Bildschirmfoto 2024-03-09 um 18 55 11" src="https://github.com/FriendsOfShopware/shopmon/assets/412142/b21bb772-4f82-4705-8e54-4216fd917613">
<img width="1234" alt="Bildschirmfoto 2024-03-09 um 18 55 22" src="https://github.com/FriendsOfShopware/shopmon/assets/412142/bd471b73-64c3-4bdf-8ad2-6ab65f1110f3">

Reply:
Added a slot for the action label and filled it in these cases